### PR TITLE
deprecated-features: Document the Deprecated features subsystem

### PR DESCRIPTION
**Pull request by dumbbell**
The list of deprecated features is also documented in the release information section.

### CODE REVIEW

**Comment by dumbbell** [comment]
I will copy the article to `v4.0` once the review process is finished.

File: docs/deprecated-features/index.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+
+## Configuring deprecated features behavior
+
+As described above, a user may decide to turn off a deprecated feature to test
+the behavior of RabbitMQ and their applications as if the feature was already
+removed.
+
+```ini
+# Try RabbitMQ as if the deprecated feature was gone.
+deprecated_features.permit.some_deprecated_feature = false
```
Comment (michaelklishin) [inline docs/deprecated-features/index.md:89]: We should list the names of such features in 3.13.x and 4.0.x.

**Review by ansd — APPROVED** [review]
Thank you for writing this docs page!

Suggestions:
1. Mention that there is also a Management UI tab for deprecated features.
2. After this PR is merged, link to the new docs from the Management UI.

File: docs/deprecated-features/index.md (hunk 1 above)
Comment (dumbbell) [reply docs/deprecated-features/index.md:89]: Ok, adding the list. Thank!

Comment (dumbbell) [reply docs/deprecated-features/index.md:95]: The list is now available in a dedicated page at `/docs/next/deprecated-features/list`.

**Comment by dumbbell** [comment]
> Suggestions:
> 
>     1. Mention that there is also a Management UI tab for deprecated features.

Done.

>     2. After this PR is merged, link to the new docs from the Management UI.

Ok, I will prepare a patch.

**Comment by dumbbell** [comment]
In fact, the list of deprecated features is global, not tied to a specific version of RabbitMQ. I will move it to `src/pages`.

**Pull request merged**